pandas/core/indexing.py

```diff
@@ -328,6 +328,106 @@ def _setitem_with_indexer(self, indexer, value):
                     take_split_path = True
                     break
 
+        def can_do_equal_len(labels, value, plane_indexer, lplane_indexer):
```

Comment:
can you make these module level functions instead?

Reply:
Well not really, But I have made them methods instead.